docs/REDUX_ACTIONS.md

```diff
@@ -42,7 +42,7 @@ When entering actions execution order:
 ![alt text](assets/login_screen.png "Login Screen")
 
 After clicking `Log in` it executes:
-- [`activePeerSet`](https://github.com/LiskHQ/lisk-hub/blob/86ab1874d89519a37d817c326c37d594f29b94eb/src/components/login/index.js#L28) with passphrase<br/>
+- [`liskAPIClientSet`](https://github.com/LiskHQ/lisk-hub/blob/86ab1874d89519a37d817c326c37d594f29b94eb/src/components/login/index.js#L28) with passphrase<br/>
```

Comment:
The link also needs to be updated to link to the version when it's called `liskAPIClientSet`
```suggestion
- [`liskAPIClientSet`](https://github.com/LiskHQ/lisk-hub/blob/d2207d4e8cab7776bf6fc3b3544afb8808866fac/src/components/login/index.js#L27) with passphrase<br/>
```